src/flows/install-happ.ts

```diff
@@ -177,12 +178,25 @@ export const setupInstance = async (client, {instanceId, agentId, dnaId, conduct
   ])
 }
 
-export const setupHolofuelBridge = async (client, {callerInstanceId}) => {
-  return client.call('admin/bridge/add', {
+export const setupHolofuelBridge = async (client, {callerInstanceId, replace}) => {
+
+  const bridgeConfig = {
     handle: 'holofuel-bridge',
     caller_id: callerInstanceId,
     callee_id: Config.holofuelId.instance,
-  })
+  }
+
+  const bridges = await client.call('admin/bridge/list', {})
+  if (bridges.find(b => _.isEqual(b, bridgeConfig))) {
+    if (replace) {
+      throw "Bridge setup with replacement not yet supported"
+    } else {
```

Comment:
Else not needed here, since we throw earlier ;)

Reply:
Sure, it's just a style preference thing. If you want to change things like this, feel free, I slightly prefer it this way with explicit if-else branches, but don't feel too strongly about it.

Reply:
I don't feel strongly about it either. Just think that this helps with readability by avoiding nesting. :)